src/nvector/trilinos/nvector_trilinos.cpp

```diff
@@ -87,6 +87,7 @@ N_Vector N_VNewEmpty_Trilinos(SUNContext sunctx)
   v->ops->nvspace           = N_VSpace_Trilinos;
   v->ops->nvgetcommunicator = N_VGetCommunicator_Trilinos;
   v->ops->nvgetlength       = N_VGetLength_Trilinos;
+  v->ops->nvgetlocallengt   = N_VGetLocalLength_Trilinos;
```

Comment:
```suggestion
  v->ops->nvgetlocallength  = N_VGetLocalLength_Trilinos;
```
(although I agree with David that any changes to this module are unnecessary)

Reply:
> Aside from one typo, I mainly question the need for this extra value in the N_Vector implementations that do not have `nvgetarraypointer` for F2003 users. I wonder if leaving this new routine as `NULL` in those implementations would be preferable so that we could ensure a segmentation fault when called.

Yes, I agree that these are not needed. I had implemented them when I thought we might make it a real op not a temporary one.